src/components/List.js

```diff
@@ -12,7 +12,11 @@ const List = ({
     buttonText,
     children,
     ...rest,
-}) => (
+}) => {
+    if(tab.length === 0) {
+        return (<div className='task-list--empty'>ToDo List Is Empty!</div>)
```

Comment:
it is good approach to add prefixes like `is` and `has` to bool values as props, but also classes. Please, go to `bulma.io`, as we really like this framework's approach. 
Also, I think, that this is misunderstanding a little the usage of blocks, elements and modifier (BEM). This element, seems to be the element of the task-list (it has no class `task-list`). Maybe more meaningful in this case, would be name: `task-list__empty-msg` ? WDYT?